2nd.py

```
@@ -0,0 +1,11 @@
+# simple_utils.py - A tiny utility library
+
+def reverse_string(text):
+    """Reverses the characters in a string."""
+    return text[::-1]
+
+def count_words(sentence):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)